js/script.js

```diff
@@ -1,46 +1,47 @@
-function Team(name, members) {
-    this.name = name;
-    this.members = members;
-}
- 
-Team.prototype.getSumOfAge = function() {
-   var sum = 0;
-   for(var i = 0; i < this.members.length; i++) {
-       sum += this.members[i].age;
-   }
-   
-  return sum;
-}
+class Team {
+    constructor(name, members) {
+        this.name = name;
+        this.members = members;
+    }
 
+    getSumOfAge() {
```

Comment:
update your summation method to map reduce

Reply:
fix